From 2370e90f34cd7b7d2e2d58d027d69abc6d729398 Mon Sep 17 00:00:00 2001
From: Andrew Boie <andrew.p.boie@intel.com>
Date: Tue, 27 Nov 2012 12:20:55 -0800
Subject: build: Conditionally include iago image creation Makefile

This file contains directives which cannot go in an Android.mk
and needs to be after they all have been included, similar to the
other Android image directives in build/core/Makefile.

Change-Id: I4c07f13a857f19052e15c562805f8f64452e4c3d
Signed-off-by: Andrew P. Boie <andrew.p.boie@intel.com>

Conflicts:
	core/cleanbuild.mk
---
 core/Makefile      |    3 +++
 core/cleanbuild.mk |    2 ++
 2 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/core/Makefile b/core/Makefile
index f91000f..37808f7 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -1087,6 +1087,9 @@ endif # BOARD_VENDORIMAGE_FILE_SYSTEM_TYPE
 ifeq ($(TARGET_USE_DISKINSTALLER),true)
 include bootable/diskinstaller/config.mk
 endif
+ifeq ($(TARGET_USE_IAGO),true)
+include bootable/iago/image.mk
+endif
 
 # -----------------------------------------------------------------
 # host tools needed to build dist and OTA packages
diff --git a/core/cleanbuild.mk b/core/cleanbuild.mk
index 0a0a2bb..5071f63 100644
--- a/core/cleanbuild.mk
+++ b/core/cleanbuild.mk
@@ -179,6 +179,7 @@ installclean_files := \
 	$(HOST_OUT)/obj/NOTICE_FILES \
 	$(HOST_OUT)/sdk \
 	$(PRODUCT_OUT)/*.img \
+	$(PRODUCT_OUT)/*.iso \
 	$(PRODUCT_OUT)/*.txt \
 	$(PRODUCT_OUT)/*.xlb \
 	$(PRODUCT_OUT)/*.zip \
@@ -197,6 +198,7 @@ installclean_files := \
 	$(PRODUCT_OUT)/System.map \
 	$(PRODUCT_OUT)/kernel \
 	$(PRODUCT_OUT)/droidboot \
+	$(PRODUCT_OUT)/iago \
 	$(PRODUCT_OUT)/obj/FAKE \
 	$(PRODUCT_OUT)/obj/EXECUTABLES/adbd_intermediates \
 	$(PRODUCT_OUT)/obj/EXECUTABLES/init_intermediates \
-- 
1.7.4.1

